lib/elixir/lib/list.ex

```diff
@@ -577,6 +577,24 @@ defmodule List do
   end
 
   @doc """
+  Returns true if `list` starts with the given `prefix` list.
```

Comment:
This is the nitpickiest of nitpicks, but could we put backticks around `true` to show it in code font? :bowtie: The rest of the PR is perfect! 💟